sdk/eventhub/test-resources.json

```diff
@@ -195,6 +203,16 @@
         "scope": "[resourceGroup().id]"
       }
     },
+    {
+      "type": "Microsoft.Authorization/roleAssignments",
+      "apiVersion": "2019-04-01-preview",
+      "name": "[guid(resourceGroup().id, deployment().name, parameters('baseName'), variables('eventHubsDataSenderRoleId'), parameters('testApplicationOid'))]",
```

Comment:
This Event Hubs Data Sender Role is not needed, because the EH Data Owner Role above already gives permission for both sending and receiving.

Reply:
Ok then the error message https://github.com/Azure/azure-sdk-for-python/pull/21715#issuecomment-1005489465 could be misleading and not indicating the underlying problem.

Reply:
I have removed it.